### DISCUSSION TITLE
how can i Contributing radix-ui?

### DISCUSSION THREAD

**Discussion by DoGukKim** [radix-ui/primitives#2292] [Help]
someone tell me more detail process?

i did forked -> remote add upstream -> pull upstream main -> set node -v .

yarn install throw error .. 🥲 what am i miss it?

**Comment by cantdocpp** [comment]
have you read the: https://github.com/radix-ui/primitives/blob/main/.github/CONTRIBUTING.md yet  ?

**Reply by DoGukKim** [reply]
i readed it! but if i install packages they throw error..!
so i cant run storybook 😅
